plugins/firebase_lumberdash/example/pubspec.yaml

```diff
@@ -4,12 +4,12 @@ version: 1.0.0
 publish_to: none
```

Comment:
nit: Just noticed that in the description field above it says lumberjack instead of lumberdash.